README.md

```diff
@@ -12,6 +12,9 @@ the FHIR store type.
 
 This codebase serves as an example of how the mCSD ITI-91 update mechanism can be implemented. It is **NOT** intended for production use.
 
+This application does NOT process, store, or handle any Personally Identifiable Information (PII).
+All data used for testing or demonstration is synthetic or non-sensitive.
```

Comment:
I'm not sure if we can state that this application does not process PII. If an mCSD directory provides for example a Practitioner this application will process this.